app/src/utils.ts

```diff
@@ -61,6 +61,11 @@ export function formatDatoKort(dato: Date) {
   }).format(dato);
 }
 
+export function leggTilDagerPåDato(dato: Date, antallDager: number) {
+  dato.setDate(dato.getDate() + antallDager);
+  return dato;
+}
+
```

Comment:
Vil ikke dette være rart på den 1. i måneden eller så?

Reply:
[Ser ut fra eksempelet til setDate](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate) at den funker med antalldager, og looper til neste mnd om man går over månedsskifte.

Sorry, jeg glemte helt av denne PRen 😅 
Nå tror jeg uansett at det ble enighet om å endre på datamodell for refusjonsperioder, slik at denne funksjonen ikke er relevant lenger...

Reply:
Åja, men da så! 😄